src/WPML_OptionsManager.php

```diff
@@ -547,59 +563,55 @@ private function displayMP3Banner() {
             $option = rand(1, 2);
             add_option('wpml_banner_version', $option);
         }
-        ?>
-        <div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;margin-top: 15px;padding: 20px;">
-            <h3><?php _e( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' );?></h3>
-            <?php
-              if ($option === 1) {
-                    $this->displayMP3BannerVersionA();
-              } else {
-                    $this->displayMP3BannerVersionB();
-              }
-            ?>
-        </div>
-        <?php
+
```

Comment:
Could you please explain why this method's approach to output construction needed to change?
If it was not necessary, could you please revert back the original method code?

Reply:
It was an artifact of the original hope/scope that banner content would be customizable.
It's back to previous method format.